getting-started/templates/systemlink-values.yaml

```diff
@@ -845,30 +845,36 @@ saltmaster:
 ## Feed configuration.
 ##
 feedservice:
-  ## Configure S3 access.
+  ## Configure file storage settings
   ##
-  s3:
-    ## Secret name for S3 credentials.
-    ##
-    secretName: "feeds-s3-credentials"
-    ## The name of the S3 bucket that the service should connect to.
-    ##
-    bucket: "systemlink-feeds"
-    ## S3 connection scheme.
-    ##
-    scheme: *s3Scheme
-    ## Set this value to connect to an external S3 instance.
-    ##
-    host: *s3Host
-    ## Set this value to connect to an S3 instance which is internal to the cluster. Ignored if host is set.
+  storage:
+    ## Storage type. Supported values: "s3".
     ##
-    service: *s3ServiceName
-    ## S3 Port
+    type: "s3"
+    ## Configure S3 access.
     ##
-    port: *s3Port
-    ## S3 Region
-    ##
-    region: *s3Region
+    s3:
+      ## Secret name for S3 credentials.
+      ##
+      secretName: "feeds-s3-credentials"
+      ## The name of the S3 bucket that the service should connect to.
+      ##
+      bucket: "systemlink-feeds"
+      ## S3 connection scheme.
+      ##
+      scheme: *s3Scheme
+      ## Set this value to connect to an external S3 instance.
+      ##
+      host: *s3Host
+      ## Set this value to connect to an S3 instance which is internal to the cluster. Ignored if host is set.
```

Comment:
Suggestion: "Set this value to connect to an internal S3 instance of a cluster. The feedservice ignores the value if the host is set."